### PR TITLE
Periodic lunch messages

### PR DESCRIPTION
**Pull request by wvdeutekom**
Added periodic messaging in this PR.

Changes:
- Added cron, now a periodic message can be sent to all subscribed channels. This is configured in the json config file
  - Every time a periodic message is sent the api is checked for joined public and private channels
  - After collection is done the lunch message for that day is sent to all those channels
- Added new lunch messages, these are now concatenated using several arrays of lunch messages:
  - `introMessages` that can be optionally prepended to any lunchmessage
  - `midLNFMessages` general lunch not found messages
  - `postLNFMessages` quotes that are appended after the mid section
-

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -3,6 +3,9 @@
     "restricted_channels": [
       "C594N2UHG",
       "C07J1HXF0"
+    ],
+    "notification_times": [
+      "0 55 11 * * *"
```
Comment (TheWolfNL) [inline config.json]: Do you also want it to trigger on saturday's and sunday's?

File: lunch/lunch.go [hunk 2]
```diff
@@ -11,15 +11,30 @@ import (
...
+		"No special lunch found, can someone insert my batteries? -BEEP-",
+		"Where is that lunch? Maybe you should try turning me off and on again",
+	}
+
+	postLNFMessages = []string{
+		`"404 Lunch not found" - Mollie monolith backend`,
+		`"There will be bread." - a bread fanatic`,
+		`"Elementary, my dear Watson. It looks like bread." - Mollie Holmes, probably.`,
+		`"Keep your friends close, but your bread closer." - Sun Tzu`,
+		`"Bread. Shaken, not stirred." - James Bond`,
```
Comment (TheWolfNL) [inline lunch/lunch.go]: Wasn't this `James Bread`?

File: lunch/lunch.go [hunk 3]
```diff
@@ -11,15 +11,30 @@ import (
...
 		"We'll always have bread.",
```
Comment (TheWolfNL) [inline lunch/lunch.go]: This could be turned into `"I'll always have bread, bread with peanutbutter." - Tjeerd`

File: main.go [hunk 4]
```diff
@@ -96,7 +97,25 @@ func main() {
...
 }
+
+func (context *AppContext) startCrons() {
+
+	cron := cron.New()
+	for _, cronTime := range context.Message.NotificationTimes {
+		fmt.Println("adding cron ", cronTime)
+		cron.AddFunc(cronTime, func() {
+			lunchMessage := context.Lunch.GetLunchMessageOfToday(true)
+			fmt.Println(lunchMessage)
```
Comment (TheWolfNL) [inline main.go]: Dev cleanup 😄

File: lunch/lunch.go (hunk 2 above)
Comment (wvdeutekom) [inline lunch/lunch.go]: I like that one

File: lunch/lunch.go (hunk 3 above)
Comment (wvdeutekom) [inline lunch/lunch.go]: I'll add yours, too :) This one is a variant on "We'll always have Paris" from Casablanca

**Comment by wvdeutekom** [comment]
🚢 🚀

**Pull request merged**